itou/archive/management/commands/notify_archive_jobseekers.py

```diff
@@ -50,24 +50,78 @@ def inactive_jobseekers_without_related_objects(inactive_since, batch_size):
     )
 
 
+def get_year_month_or_none(date=None):
+    if not date:
+        return None
+
+    if isinstance(date, datetime.datetime):
+        return timezone.localdate(date).replace(day=1)
+
+    return date.replace(day=1)
+
+
 def anonymized_jobseeker(user):
-    kwargs = {
-        "date_joined": timezone.localdate(user.date_joined).replace(day=1),
-        "first_login": timezone.localdate(user.first_login).replace(day=1) if user.first_login else None,
-        "last_login": timezone.localdate(user.last_login).replace(day=1) if user.last_login else None,
-        "user_signup_kind": getattr(user.created_by, "kind", None),
-        "department": user.department,
-        "title": user.title,
-        "identity_provider": user.identity_provider,
-        "had_pole_emploi_id": bool(user.jobseeker_profile.pole_emploi_id),
-        "had_nir": bool(user.jobseeker_profile.nir),
-        "lack_of_nir_reason": user.jobseeker_profile.lack_of_nir_reason,
-        "nir_sex": user.jobseeker_profile.nir[0] if user.jobseeker_profile.nir else None,
-        "nir_year": user.jobseeker_profile.nir[1:3] if user.jobseeker_profile.nir else None,
-        "birth_year": user.jobseeker_profile.birthdate.year if user.jobseeker_profile.birthdate else None,
-    }
-
-    return ArchivedJobSeeker(**kwargs)
+    job_applications = JobApplication.objects.filter(job_seeker=user).aggregate(
+        count_accepted_applications=Count("id", filter=Q(state=JobApplicationState.ACCEPTED)),
+        count_IAE_applications=Count("id", filter=Q(to_company__kind__in=CompanyKind.siae_kinds())),
+        count_total_applications=Count("id"),
+    )
+    return ArchivedJobSeeker(
+        date_joined=get_year_month_or_none(user.date_joined),
+        first_login=get_year_month_or_none(user.first_login),
+        last_login=get_year_month_or_none(user.last_login),
+        user_signup_kind=getattr(user.created_by, "kind", None),
+        department=user.department,
+        title=user.title,
+        identity_provider=user.identity_provider,
+        had_pole_emploi_id=bool(user.jobseeker_profile.pole_emploi_id),
+        had_nir=bool(user.jobseeker_profile.nir),
+        lack_of_nir_reason=user.jobseeker_profile.lack_of_nir_reason,
+        nir_sex=user.jobseeker_profile.nir[0] if user.jobseeker_profile.nir else None,
+        nir_year=user.jobseeker_profile.nir[1:3] if user.jobseeker_profile.nir else None,
+        birth_year=user.jobseeker_profile.birthdate.year if user.jobseeker_profile.birthdate else None,
+        count_accepted_applications=job_applications["count_accepted_applications"],
+        count_IAE_applications=job_applications["count_IAE_applications"],
+        count_total_applications=job_applications["count_total_applications"],
+    )
+
+
+def anonymized_jobapplication(obj):
+    return ArchivedApplication(
+        job_seeker_birth_year=obj.job_seeker.jobseeker_profile.birthdate.year
+        if obj.job_seeker.jobseeker_profile.birthdate
+        else None,
```

Comment:
nit: je trouve
```suggestion
        job_seeker_birth_year=(
            obj.job_seeker.jobseeker_profile.birthdate.year
            if obj.job_seeker.jobseeker_profile.birthdate
            else None
        ),
```
plus lisible (et pareil pour les autres).